web/src/lib/utils/asset-utils.ts

```diff
@@ -131,11 +131,35 @@ export function getFileMimeType(file: File): string {
 		dng: 'image/dng',
 		heic: 'image/heic',
 		heif: 'image/heif',
+		avif: 'image/avif',
 		insp: 'image/jpeg',
 		insv: 'video/mp4',
 		nef: 'image/x-nikon-nef',
 		raf: 'image/x-fuji-raf',
-		srw: 'image/x-samsung-srw'
+		srw: 'image/x-samsung-srw',
+		crw: 'image/x-canon-crw',
+		cr2: 'image/x-canon-cr2',
+		cr3: 'image/x-canon-cr3',
+		erf: 'image/x-epson-erf',
+		raf: 'image/x-fuji-raf',
+		dcr: 'image/x-kodak-dcr',
+		k25: 'image/x-kodak-k25',
+		kdc: 'image/x-kodak-kdc',
+		mrw: 'image/x-minolta-mrw',
+		orf: 'image/x-olympus-orf',
+		raw: 'image/x-panasonic-raw',
```

Comment:
I have some `rw2` files that, as far as I understand, are the same `x-panasonic-raw` files. Could this extension be added as well?

Reply:
I looked into this. .rw2 files are a bit different than .raw files, and libraw does not presently support them according to https://www.libraw.org/node/2710 . The format wasn't on the digikam or gnome-raw-thumbnailer list of formats supported by libraw.